specification/communication/data-plane/Microsoft.CommunicationServicesAdministration/preview/2020-07-20-preview1/examples/GetLocationOptionsAsync.json

```diff
@@ -0,0 +1,71 @@
+{
+  "parameters": {
+    "api-version": "2020-07-20-preview1",
+    "countryCode": "US",
+    "endpoint": "https://resourcename.region.dev.communications.azure.net/",
+    "phonePlanGroupId": "671ee064-662f-4c3b-82a9-af2ab200dd5c",
+    "phonePlanId": "27b53eec-8ff4-4070-8900-fbeaabfd158a"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "locationOptions": {
+          "labelId": "state",
+          "labelName": "State",
+          "options": [
+            {
+              "name": "AK",
+              "value": "AK",
+              "locationOptions": [
+                {
+                  "labelId": "city",
+                  "labelName": "City",
+                  "options": [
+                    {
+                      "name": "Anchorage",
+                      "value": "NOAM-US-AK-AN",
+                      "locationOptions": null
+                    }
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "AL",
+              "value": "AL",
+              "locationOptions": [
+                {
+                  "labelId": "city",
+                  "labelName": "City",
+                  "options": [
+                    {
+                      "name": "Birmingham",
+                      "value": "NOAM-US-AL-BI",
+                      "locationOptions": null
```

Comment:
let's remove the `null`s. Same for other responses.

Reply:
Done.